projects/dxilconv/test/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ add_lit_target("check-dxilconv" "Running lit suite dxilconv"
           ${CMAKE_CURRENT_SOURCE_DIR}/taef
           PARAMS
                  dxilconv_site_config=${CMAKE_CURRENT_BINARY_DIR}/taef/lit.site.cfg
+                 taef_filter="/select:\"@Architecture='${DXC_BUILD_ARCH}'\""
```

Comment:
Please use `CMAKE_C_COMPILER_ARCHITECTURE_ID` instead of `DXC_BUILD_ARCH`. We should remove `DXC_BUILD_ARCH` as it isn't needed (see: #4156).